appcelerator/titanium_mobile
# [TIMOB-24930] Fix build process erroring when NDK path not set

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24930

- Only check for existence of this.androidInfo.ndk

## Verification

1. Remove the NDK path from your ti config, ` [appc] ti config --remove android.ndkPath` or hand edit the file
2. Build an Android module
     - Should error with `[ERROR] Unable to find a suitable installed Android NDK.`
